integration/nodejs/express.mdx

```diff
@@ -148,8 +148,43 @@ app.get('/admin/hello', async (req: Request, res: Response) => {
 });
 ```
 
-## PKCE and Client Secrets
+## Authentication Flows
 
-Civic Auth uses [**PKCE** (Proof Key for Code Exchange)](https://oauth.net/2/pkce/), to protect users and clients from unauthorized access to user information. This, alongside domain registration for apps in production environments, mean that you don't need to provide a client secret in your backend.
+Civic Auth supports multiple OAuth 2.0 authentication methods to provide maximum security for different application architectures.
 
-When using the Civic Auth SDK, PKCE is handled entirely by the library.
+<Info>
```

Comment:
This info section is  fine, but I would leave out the rest (159-186). especially since client-secret is only available in "contact civic" mode right now.  It adds confusion - reading this I don't immediately know what to do . do I have to contact civic or not?